config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
*nod* yeah I don't view this as mergeable yet (hopefully the state of the specs makes that obvious). Just approaching mergeable. Hopefully one more weekend of work on my end will get us to 99% ready